src/odemis/driver/test/technolution_test.py

```diff
@@ -1735,7 +1738,7 @@ def image_received(self, *args):
         """
         image = args[1]
         # Check resolution of received image
-        if image.shape != self.dataContent2Resolution(self.MPPC.dataContent.value):
+        if image.shape != self.data_content2_resolution(self.mppc.dataContent.value):
```

Comment:
Would it not be nicer to drop the 2 and just change it to a word?
Like this: `self.data_content_to_resolution`

Reply:
agreed, your name is better, fixed it!